sci-physics/lhapdf/lhapdf-6.3.0-r1.ebuild

```diff
@@ -0,0 +1,82 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..9} )
+DISTUTILS_USE_SETUPTOOLS=no
+DISTUTILS_SINGLE_IMPL=1
+inherit distutils-r1
+
+MY_PV=$(ver_cut 1-3)
+MY_PF=LHAPDF-${MY_PV}
+
+DESCRIPTION="Les Houches Parton Density Function unified library"
+HOMEPAGE="http://lhapdf.hepforge.org/"
+SRC_URI="http://www.hepforge.org/archive/lhapdf/${MY_PF}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="doc examples python"
+REQUIRED_USE="python? ( ${PYTHON_REQUIRED_USE} )"
+
+RDEPEND="
+	dev-libs/boost:=
+	python? ( ${PYTHON_DEPS} )"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	doc? (
+		app-doc/doxygen
+		dev-texlive/texlive-bibtexextra
+		dev-texlive/texlive-fontsextra
+		dev-texlive/texlive-fontutils
+		dev-texlive/texlive-latex
+		dev-texlive/texlive-latexextra
+	)"
+
+S="${WORKDIR}/${MY_PF}"
+
+src_configure() {
+	CONFIG_SHELL="${EPREFIX}/bin/bash" \
+	econf \
+		--disable-static \
+		$(use_enable python)
+
+	if use python; then
+		cd "${S}"/wrappers/python || die
+		distutils-r1_src_prepare
```

Comment:
Because we are switching to single target we can probably simplify this ebuild by:
- dropping all the `if use python` stuff and let `$(use_enable python)` take care of things.
- switching from distutils-r1 eclass to python-single-r1 eclass
Similar to the 6.5.X ebuilds.

Or maybe even enable python unconditionally since this is what the 6.5.X ebuilds seem to do.